proposals/NNNN-Change_ATF_test_reports_folder_structure.md

```diff
@@ -0,0 +1,55 @@
+# Change ATF test reports folder structure
+
+* Proposal: [SDL-NNNN](NNNN-Change_ATF_test_reports_folder_structure.md)
+* Author: [Irina Getmanets](https://github.com/GetmanetsIrina)
+* Status: **Awaiting review**
+* Impacted Platforms: ATF
+
+## Introduction
+
+For now ATF reports folder has inconvenient nested structure. ATF creates structure of folders from relative path to script, e.g. runned script fom test_scripts/API/Navidation/Subscriptions/.
```

Comment:
@dboltovskyi @GetmanetsIrina It is too big and to detailed for introduction. 
Introduction must contain hight level description of changes.
Like : 
Main output of ATF is test reports. 
ATF provide such reports : 
  - Console logs
  - SDL logs
  - Transport  logs
  - Detailed report

This proposal is about creating clear and useful structure of test scripts reports. 
Multiple running of test script should not override old reports. 
Structure of reports should be clear for easy searching certain report.

Reply:
@LuxoftAKutsan , please find changes in commit 14e20e522aa7592aebf356eec293d24797f7802e.